packages/now-cli/src/util/credit-cards.js

```diff
@@ -38,7 +38,7 @@ export default class CreditCards extends Now {
     return true;
   }
 
-  add(card) {
+  async add(card) {
```

Comment:
Why was this changed? Should we refactor to remove the `new Promise()` below?

Reply:
I changed it for uniformity and forgot to remove the promise... Fixed